### PR TITLE
Wip redirect to img

### PR DESCRIPTION
**Pull request by realAndrewCohn**
Instead of creating UUIDs for each student, the app now creates a new image for each user session and ~deletes the last 4000 records once we exceed 5000 in db~ deletes any images older than 60 days. If this looks good, I'll update the student facing instructions

### CODE REVIEW

File: app/controllers/images_controller.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ def index
   end
 
   def show
-    @image = ImageCreator.new(uuid: params[:uuid]).find_or_create_image
```
Comment (nikymorg) [inline app/controllers/images_controller.rb]: delete image creator if you aren't using it

File: app/controllers/welcome_controller.rb [hunk 2]
```diff
@@ -3,14 +3,14 @@ class WelcomeController < ActionController::Base
   def index
     respond_to do |format|
 
-      format.html { 
-        session[:image_id] ||= SecureRandom.uuid
+      format.html {
+        session[:image_id] ||= Image.create.id
```
Comment (nikymorg) [inline app/controllers/welcome_controller.rb]: Check if image exists and reset if it doesn't. Instructors will probably have cached images that will get deleted.

Comment (realAndrewCohn) [reply app/controllers/welcome_controller.rb]: updated

File: app/models/image.rb [hunk 3]
```diff
@@ -5,4 +5,30 @@ class Image < ApplicationRecord
...
+  before_create do |img|
+    unless img.url && img.name
+      rand_details = IMAGES.sample
+      img.url = rand_details[:url]
+      img.name = rand_details[:name]
+      img.comments.build(content: 'first comment!')
+    end
+  end
+
+  after_create do |img|
```
Comment (realAndrewCohn) [inline app/models/image.rb]: also if anyone has a suggestion for a better way to handle deleting old records pls lmk

Comment (realAndrewCohn) [reply app/models/image.rb]: Way easier solution: just delete records older than 60 days

**Pull request merged**